R/build-quarto-articles.R

```diff
@@ -1,5 +1,8 @@
 build_quarto_articles <- function(pkg = ".", article = NULL, quiet = TRUE) {
   pkg <- as_pkgdown(pkg)
+  if (pkg$bs_version < 5) {
+    cli::cli_abort("Quarto articles require Bootstrap 5.", call = NULL)
```

Comment:
maybe also link to https://pkgdown.r-lib.org/articles/customise.html#getting-started